pkgs/native_assets_cli/test/api/build_output_test.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:test/test.dart';
+
+void main() {
+  late Uri tempUri;
+
+  setUp(() async {
+    tempUri = (await Directory.systemTemp.createTemp()).uri;
+  });
+
+  tearDown(() async {
+    await Directory.fromUri(tempUri).delete(recursive: true);
+  });
+
+  test('BuildOutput constructor', () {
+    BuildOutput(
```

Comment:
What does this test?

Reply:
That calling the constructor works. (It's a forwarding constructor in the API.)